src/components/BackgroundImage.js

```diff
@@ -1,8 +1,16 @@
+//@flow
 import React, { Component } from 'react';
 import { Image, StyleSheet } from 'react-native';
 import Globals from '../Globals';
 
+import type { Children } from 'react';
+
+type Props = {
+  children?: Children
+}
+
 class BackgroundImage extends Component {
+  props: Props;
```

Comment:
Yeah .. I think it's better ... tonight I'm gonna adjust it. Tks!! I've never used props/state like this ...